packages/gateway/src/features/registration/fhir-builders.ts

```diff
@@ -870,12 +870,16 @@ const builders: IFieldBuilders = {
       }
     }
   },
-  createdAt: (fhirBundle, fieldValue) => {
+  createdAt: (fhirBundle, fieldValue, context) => {
     if (!fhirBundle.meta) {
       fhirBundle.meta = {}
     }
     fhirBundle.meta.lastUpdated = fieldValue
     fhirBundle.entry[0].resource.date = fieldValue
+
+    const taskResource = selectOrCreateTaskRefResource(fhirBundle, context)
```

Comment:
In `createBirthRegistration` the submitted `details.createdAt` value is now used as the `lastModified` value for the first task in the declaration's task history